src/sentry/scripts/similarity/index.lua

```diff
@@ -195,28 +236,6 @@ local function redis_hgetall_response_to_table(response, value_type)
 end
 
 
--- Generic Configuration
-
-local configuration_parser = build_argument_parser({
-    {"timestamp", parse_integer},
-    {"namespace", identity},
-    {"bands", parse_integer},
-    {"interval", parse_integer},
-    {"retention", parse_integer},  -- how many previous intervals to store (does not include current interval)
-    {"scope", function (value)
-        assert(value ~= nil)
-        return value
-    end}
-})
-
-local function takes_configuration(command)
```

Comment:
The first commit doesn't remove this function (which screws up all the indentation from there on out), I'd recommend reviewing that one separately since it's really the meat of this change.